tests/dataset_tests/audio-cats-and-dogs/AudioCatsAndDogsTest.cpp

```diff
@@ -36,23 +36,33 @@ TEST_F(AudioCatsAndDogsTest, loadData)
     ASSERT_EQ(data->isValid(), 0);
 }
 
-TEST_F(AudioCatsAndDogsTest, DISABLED_trainNeuralNetwork)
+TEST_F(AudioCatsAndDogsTest, DISABLED_trainBestNeuralNetwork)
 {
 
     StraightforwardNeuralNetwork neuralNetwork({
         Input(sizeOfOneData),
-        MaxPooling(100),
-        Recurrence(50),
-        FullyConnected(2)
+        MaxPooling(96),
+        FullyConnected(10),
+        FullyConnected(2, activation::identity, Softmax())
```

Comment:
If you does not use activation between two FullyConnected, is pointless use both. You could leave two neurons and add activation

Reply:
Maybe for next commits, it will be a 'good idea' use an activation internally

Reply:
There is a default sigmoid activation on the FullyConnected layer.